cmd/bee/cmd/start.go

```diff
@@ -466,7 +466,7 @@ func (c *command) configureSigner(cmd *cobra.Command, logger log.Logger) (config
 		}
 	} else {
 		logger.Warning("clef is not enabled; portability and security of your keys is sub optimal")
-		swarmPrivateKey, _, err := keystore.Key("swarm", password)
+		swarmPrivateKey, _, err := keystore.Key("swarm", password, crypto.GenerateSecp256k1Key, crypto.EncodeSecp256k1PrivateKey, crypto.DecodeSecp256k1PrivateKey)
```

Comment:
nitpick: it's not in scope of this PR, but enumerating all key names would be neater